getting_started/HowTo.rst

```diff
@@ -0,0 +1,114 @@
+How to
+===============
+
+How to download the game
+************************
+
+To be able to download the game you have to be in the testing groups or have buy the game
+
+#. Download itch.io launcher at this link :  https://itch.io/app
+
+#. Once you have it install , Create a Itch account
+
+#. Go on the link we gave you and add the password on it
+
+#. Congrats you have access to the itch.io private page of NanosWorld
+
+#. Select the client you want (Be carefull they are multiple client available during the testing phase)
+
+#. Be carefull they are also the server file
+
+.. image:: ../images/HowToDownload.png
+   :width: 600
+   :alt: Overview of the launcher
+
+
+
+
+How to launch the game
+**********************
+
+#. After you have download the game go into the created folder
+
+#. Click on NanosWorld.exe
+
+.. image:: ../images/HowToLaunchTheGame.png
+   :width: 600
+   :alt: Overview of the game folder
+
+
+
+How to join a server
+********************
+
+#. Click on the play button and select the desired server
+
+the server are ranked this way :
+
+Servername , Gamemode , Maps , server slot , ping
+
+
+How to make a server
+********************
+
+#. After dowloading the server file
+
+#. Go into the server folder it will looks like this :
+
+.. image:: ../images/HowToLaunchTheServer.png
+   :width: 400
+   :alt: Overview of the Server folder
+
+#. Press NanosworldServer.exe to launch the server .
+
+
+How to modify a config server file
+**********************************
+
+.. image:: ../images/HowToModifyServerConfig.png
+   :width: 600
+   :alt: Overview of the Config file
+
+
+
+
+How to add package to my server
+*******************************
+
+Download or make gamemode and put it into the package folder in you're server
```

Comment:
does the following sentence make sense?

"folder in you are server" 

folder in I am server?

two different words: "**You Are**" and "**Your**" two different meanings

as well for "**They Are**" and "**There**" two different words and completely different meanings

```suggestion
Download or make gamemode and put it into the package folder in your server
```